test/api/c/CMakeLists.txt

```diff
@@ -0,0 +1,118 @@
+###############################################################################
+# Top contributors (to current version):
+#   Aina Niemetz
+#
+# This file is part of the cvc5 project.
+#
+# Copyright (c) 2009-2024 by the authors listed in the file AUTHORS
+# in the top-level source directory and their institutional affiliations.
+# All rights reserved.  See the file COPYING in the top-level source
+# directory for licensing information.
+# #############################################################################
+#
+# The build system configuration.
+##
+
+include_directories(.)
```

Comment:
Is this even needed? If yes, this should be done via the target_include_... below.

Reply:
It seems that this is indeed not needed.